docs/cloud-tutorials/database-management.md

```diff
@@ -58,12 +58,24 @@ database:
 To run your migrations locally, run `npx dbos migrate` or `npx dbos rollback`.
 
 When you [deploy](./application-management.md#registering-and-deploying-applications) an application to DBOS Cloud it runs `npx dbos migrate` to apply all schema changes before starting your application or updating its code.
-When you [roll back](./application-management.md#rolling-back-application-databases) an application on DBOS Cloud, it runs `npx dbos rollback` to roll back schema changes before updating your application's code.
 
 :::info
 Be careful making breaking schema changes such as deleting or renaming a column&#8212;they may break active workflows running on a previous application version.
 :::
 
+Sometimes, it may be necessary to manually run migration commands such as `npx dbos rollback` on a cloud database, for example to recover from a schema migration failure.
```

Comment:
```suggestion
Sometimes, it may be necessary to manually undo schema changes such as running `npx dbos rollback` on a cloud database, for example to recover from a schema migration failure.
```